proposals/route-blinding.md

```diff
@@ -0,0 +1,285 @@
+# Route Blinding
+
+## Table of Contents
+
+* [Proposal](#proposal)
+  * [Introduction](#introduction)
+  * [Overview](#overview)
+  * [Notations](#notations)
+  * [Requirements](#requirements)
+  * [Encrypted recipient data](#encrypted-recipient-data)
+  * [Creating a blinded route](#creating-a-blinded-route)
+  * [Sending to a blinded route](#sending-to-a-blinded-route)
+  * [Receiving from a blinded route](#receiving-from-a-blinded-route)
+  * [Sample flow](#sample-flow)
+  * [Unblinding channels via fee probing](#unblinding-channels-via-fee-probing)
+* [Tips and Tricks](#tips-and-tricks)
+  * [Recipient pays fees](#recipient-pays-fees)
+  * [Dummy hops](#dummy-hops)
+  * [Wallets and unannounced channels](#wallets-and-unannounced-channels)
+  * [Blinded trampoline route](#blinded-trampoline-route)
+* [FAQ](#faq)
+  * [Why not use rendezvous](#why-not-use-rendezvous)
+  * [Why not use HORNET](#why-not-use-hornet)
+
+## Proposal
+
+### Introduction
+
+Route blinding is a lightweight technique to provide recipient anonymity by blinding an arbitrary
```

Comment:
I think this is already quite covered in the requirements section, isn't it?
I think what you mention are things reviewers should think about when evaluating the proposal (trying to attack it, and see if it's broken) rather than things that should be in the document itself.
I updated the PR description to detail these.